### PR TITLE
Update Ldap.php to avoid warnings

### PR DESCRIPTION
**Pull request by Vincent0Lourdin**
Change some characters in the filter's concatenation string (lines 73  and 74) in order to get rid of a lot of warning on the web interface when displaying all the users on the index page, HR side.

### CODE REVIEW

**Comment by prytoegrian** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Libertempo/web/pullRequest?prid=6401132&bid=20751675)

File: App/Libraries/Ldap.php [hunk 1]
```diff
@@ -70,8 +70,8 @@ private function getInfosUser($nom)
 
     public function getEmailUser($login)
     {
-        $filter = "(&(" . $this->configuration['attrLogin'] . "=" . $login . ")
-                    (" . $this->configuration['attrFiltre'] . "=" . $this->configuration['filtre'] . "))";
+        $filter = "(&(" . $this->configuration['attrLogin'] . "=" . $login . ")(" .
```
Comment (prytoegrian) [inline App/Libraries/Ldap.php:73]: Codacy found an issue: [Whitespace found at end of line](https://app.codacy.com/manual/Libertempo/web/pullRequest?prid=6401132)

Comment (Vincent0Lourdin) [reply App/Libraries/Ldap.php]: Commit edited.

**Review by wouldsmina — APPROVED** [review]
Merci pour ta contribution.

**Review by wouldsmina — APPROVED** [review]

**Pull request merged**